spec/menufactory_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'menufactory'
+require 'menu'
+require 'dish'
+
+describe MenuFactory do
+
+  let(:list) { [{:name=>:sushi, :price=>5},
+                {:name=>:burger, :price=>3},
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)